Make PatternFly a peer dependency

riccardo-forina opened the pull request:
PatternFly is listed as a direct dependency, which can lead to multiple versions of PatternFly being used by a project that depends on uniforms-patternfly.

By declaring it as a peer dependency fixes this.

wtrocki commented:
@riccardo-forina Nice to see you. Making peer dependency will make things nice but also we will need some good dependency range. Our team weren't that familiar of what semver/rules that patternfly does for their releases. 

If there is some semver range that breaking changes are not introduced we can use range. 
I think that might be separate PR as this is pretty straightforward change.


@riccardo-forina do you mind updating documentation?

wtrocki commented:
@ljmotta FYI - this will affect your team as well (in best case scenario it will reduce bundle size, in worst it might require adding patternfly to dependencies in your projects.)

riccardo-forina commented:
I fixed the example app exploding (it was actually a parcel thing, not sure why it worked before) and updated the dependency to specify a minimum version for PF.

It should be ok now I think.

wtrocki commented:
Amazing. I will verify and release new version tonight.
Thank you so much for contribution
